### PR TITLE
Update code sizes

### PR DESCRIPTION
**Pull request by yourslab**
I calculated code sizes after adding version numbers in PR #53 

**Note: I am always rounding up.**

O0:
size -t libcoverity_analysis.a
```
   text    data     bss     dec     hex filename
  13577       0       0   13577    3509 core_mqtt.c.obj (ex libcoverity_analysis.a)
   5979       0       0    5979    175b core_mqtt_state.c.obj (ex libcoverity_analysis.a)
  12203       0       0   12203    2fab core_mqtt_serializer.c.obj (ex libcoverity_analysis.a)
  31759       0       0   31759    7c0f (TOTALS)
```
O1:
size -t libcoverity_analysis.a
```
   text    data     bss     dec     hex filename
   5145       0       0    5145    1419 core_mqtt.c.obj (ex libcoverity_analysis.a)
   2236       0       0    2236     8bc core_mqtt_state.c.obj (ex libcoverity_analysis.a)
   4213       0       0    4213    1075 core_mqtt_serializer.c.obj (ex libcoverity_analysis.a)
  11594       0       0   11594    2d4a (TOTALS)
```
Os:
size -t libcoverity_analysis.a
```
   text    data     bss     dec     hex filename
   4624       0       0    4624    1210 core_mqtt.c.obj (ex libcoverity_analysis.a)
   1779       0       0    1779     6f3 core_mqtt_state.c.obj (ex libcoverity_analysis.a)
   3772       0       0    3772     ebc core_mqtt_serializer.c.obj (ex libcoverity_analysis.a)
  10175       0       0   10175    27bf (TOTALS)
```

### CODE REVIEW

File: docs/doxygen/pages.txt [hunk 1]
```diff
@@ -29,23 +29,23 @@ The configurations for memory estimation are defined <a href="https://github.com
     </tr>
     <tr>
         <td>core_mqtt.c</td>
-        <td>3.3K</td>
-        <td>2.9K</td>
+        <td>6.9K</td>
```
Comment (aggarw13) [inline docs/doxygen/pages.txt:32]: This can be 6.8K as `6.8 * 1024 = 6963.2` is still larger than the `6851` value obtained from your calculations

Comment (yourslab) [reply docs/doxygen/pages.txt:32]: 6.7K works too :)

**Comment by aggarw13** [comment]
https://github.com/FreeRTOS/coreMQTT/blob/ca37fab721c8a0fcd042c2173ffde1514505649e/docs/doxygen/pages.txt#L23
Can you fix this line to not mention "LTS RC1"?

File: docs/doxygen/pages.dox [hunk 2]
```diff
@@ -20,32 +20,37 @@ The configurations for memory estimation are defined <a href="https://github.com
 
 <table>
     <tr>
-        <td colspan="3"><b>Code Size of MQTT LTS rc1(example generated with GCC for ARM Cortex-M)</b></td>
+        <td colspan="3"><b>Code Size of coreMQTT (example generated with GCC for ARM Cortex-M)</b></td>
```
Comment (aggarw13) [inline docs/doxygen/pages.dox:23]:
```suggestion
        <td colspan="3"><b>Code Size of coreMQTT library files (example generated with GCC for ARM Cortex-M)</b></td>
```

Comment (aggarw13) [reply docs/doxygen/pages.dox:23]: Would be good to hyperlink the ARM GCC toolchain

Comment (aggarw13) [reply docs/doxygen/pages.dox:23]: Also, add a statement about these numbers being rounded up from their decimal values.

File: docs/doxygen/pages.dox [hunk 3]
```diff
@@ -20,32 +20,37 @@ The configurations for memory estimation are defined <a href="https://github.com
...
-        <td>1.6K</td>
+        <td>5.9K</td>
+        <td>2.2K</td>
+        <td>1.8K</td>
     </tr>
     <tr>
         <td>core_mqtt_serializer.c</td>
-        <td>3.5K</td>
-        <td>2.9K</td>
+        <td>12.0K</td>
```
Comment (aggarw13) [inline docs/doxygen/pages.dox:45]:
```suggestion
        <td>12K</td>
```

File: docs/doxygen/pages.dox [hunk 4]
```diff
@@ -20,32 +20,37 @@ The configurations for memory estimation are defined <a href="https://github.com
...
+        <td>4.2K</td>
+        <td>3.7K</td>
     </tr>
     <tr>
         <td><b>Total estimates</b></td>
-        <td>8.9K</td>
-        <td>7.4K</td>
+        <td>31.2K</td>
```
Comment (aggarw13) [inline docs/doxygen/pages.dox:51]: `31.1 * 1024 = 31846`
```suggestion
        <td>31.1K</td>
```

File: docs/doxygen/pages.dox [hunk 5]
```diff
@@ -20,7 +20,7 @@ The configurations for memory estimation are defined <a href="https://github.com
 
 <table>
     <tr>
-        <td colspan="3"><b>Code Size of coreMQTT (example generated with GCC for ARM Cortex-M)</b></td>
+        <td colspan="4"><center><b>Code Size of coreMQTT library files (sizes generated with [GCC for ARM Cortex-M toolchain 20191025](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads))</b></center></td>
```
Comment (aggarw13) [inline docs/doxygen/pages.dox:23]: This link points to the latest release of the toolchain. Could you provide the link to 2019-q4 release?

**Review by aggarw13 — APPROVED** [review]

**Review by aggarw13 — APPROVED** [review]

**Review by gkwicker — APPROVED** [review]

**Pull request merged**